### PR TITLE
Updating multimarkdown formula

### PR DESCRIPTION
**Pull request by amirmasoudabdol**
Not sure if I should do more or not!

### CODE REVIEW

**Comment by mistydemeo** [comment]
Looks like this installs a Windows bat file?

```
 * Non-executables were installed to "/usr/local/Cellar/multimarkdown/4.7.1/bin"
The offending files are:
  /usr/local/Cellar/multimarkdown/4.7.1/bin/markdown.bat
```

File: Library/Formula/multimarkdown.rb [hunk 1]
```diff
@@ -3,7 +3,7 @@
 class Multimarkdown < Formula
   homepage 'http://fletcherpenney.net/multimarkdown/'
   # Use git tag instead of the tarball to get submodules
-  url 'https://github.com/fletcher/MultiMarkdown-4.git', :tag => '4.6'
+  url 'https://github.com/fletcher/MultiMarkdown-4.git', :tag => '4.7.1'
```
Comment (xu-cheng) [inline Library/Formula/multimarkdown.rb]: Can you specify `:revision as well? Thanks.

**Comment by amirmasoudabdol** [comment]
@mistydemeo I really wonder how to do it. I looked around and I found that I should use something like this: `rm Dir["*.bat", "bin/*.bat"]` but when I add this right after `install` and install the new formula, it doesn't remove the `bat` file. I've try to put it at the end of `install` function but no success. Could you point me to the right documentation? 

Thanks,
Amir.

File: Library/Formula/multimarkdown.rb (hunk 1 above)
Comment (amirmasoudabdol) [inline Library/Formula/multimarkdown.rb]: Could you explain more or point me to the right docs, I am not sure if I know what to do. I found out that there are two types of `revision`. I can add it after `head` like `revision 1` which I guess force recompile. Or use `:revision` on url which I don't what it does.

Thanks,
Amir.

**Pull request closed without merging**